### PR TITLE
Make initContainers of terraform-{apply,destroy}-job configurable (#158)

### PR DESCRIPTION
**Pull request by wangwang**
Make initContainers of terraform-{apply,destroy}-job configurable (#158)
Bump chart version

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= oamdev/terraform-controller:0.2.8
+IMG ?= oamdev/terraform-controller:0.2.14
```
Comment (zzxwill) [inline Makefile:3]: Please keep this line as it is. I am writing a script to automatically replacing 0.2.8 to its latest tag.

File: chart/values.yaml [hunk 2]
```diff
@@ -2,9 +2,11 @@ replicaCount: 1
 
 image:
   repository: oamdev/terraform-controller
-  tag: 0.2.8
+  tag: 0.2.14
```
Comment (zzxwill) [inline chart/values.yaml:5]: Please keep this line as it is. I am writing a script to automatically replacing 0.2.8 to its latest tag.

**Comment by zzxwill** [comment]
This feature is in great need! Thanks.

File: Makefile (hunk 1 above)
Comment (wangwang) [reply Makefile]: updated

File: chart/values.yaml (hunk 2 above)
Comment (wangwang) [reply chart/values.yaml]: done

**Pull request merged**